src/App.js

```diff
@@ -1,5 +1,37 @@
+import { MissionUtils } from '@woowacourse/mission-utils'
+import { MESSAGE } from './constants/message.js'
+import {
+	inputCarsNameAndValidate,
+	inputTryCountAndValidate,
+	printResult,
+	racingResult,
+	winner,
+} from './utils/index.js'
+
 class App {
-  async play() {}
+	async play() {
+		//자동차 이름 입력
+		MissionUtils.Console.print(MESSAGE.START)
+		const carsNameSplit = await inputCarsNameAndValidate()
+		let carsInfo = new Map(carsNameSplit.map(carName => [carName, 0]))
+
+		//시도할 횟수 입력
+		MissionUtils.Console.print(MESSAGE.TRY_COUNT)
+		const tryCount = await inputTryCountAndValidate()
+
+		//레이싱 결과 출력
+		MissionUtils.Console.print(MESSAGE.RESULT)
+
+		for (let i = 0; i < tryCount; i++) {
+			racingResult(carsInfo)
+			printResult(carsInfo)
+		}
+		const printWinner = winner(carsInfo)
```

Comment:
```suggestion
		const printWinner = getWinners(carsInfo)
```
`winner`는 함수명보다는 정적인 변수명에 어울리는 것 같아서 `getWinners` 이런식으로 바꿔봐도 좋을 것 같아요!